mibstore functions are no longer supported in the X server.  
This patch removes them from this driver, following the pattern in
   http://patches.openembedded.org/patch/46133/

This checkin shows when/where the changes to the X server were made
that deleted the header mibstore.h and mentions the reasons.
   http://lists.x.org/archives/xorg-devel/2012-September/033575.html

Upstream-Status: Pending
Signed-off-by: Gary Thomas <gary@mlbassoc.com> 
--

Index: xserver-xorg-video-imx-viv-12.09.01/src/vivante_util/vivante_common.h
===================================================================
--- xserver-xorg-video-imx-viv-12.09.01.orig/src/vivante_util/vivante_common.h
+++ xserver-xorg-video-imx-viv-12.09.01/src/vivante_util/vivante_common.h
@@ -48,7 +48,6 @@ extern "C" {
 
 
 #include "mipointer.h"
-#include "mibstore.h"
 #include "micmap.h"
 #include "mipointrst.h"
 #include "inputstr.h"
Index: xserver-xorg-video-imx-viv-12.09.01/src/vivante_fbdev/vivante_fbdev_driver.c
===================================================================
--- xserver-xorg-video-imx-viv-12.09.01.orig/src/vivante_fbdev/vivante_fbdev_driver.c
+++ xserver-xorg-video-imx-viv-12.09.01/src/vivante_fbdev/vivante_fbdev_driver.c
@@ -746,7 +746,6 @@ VivScreenInit(SCREEN_INIT_ARGS_DECL) {
 
 
     xf86SetBlackWhitePixels(pScreen);
-    miInitializeBackingStore(pScreen);
     xf86SetBackingStore(pScreen);
 
     pScrn->vtSema = TRUE;
